src/main/components/Browse/Browse.js

```diff
@@ -105,15 +226,33 @@ const DASHBOARD_QUERY = gql`
         id
         title
         description
+        jiraId
+        libraryIds
+        sampleIds
+        project
       }
     }
   }
 `;
 
+const GRAPHQL_COLUMNS = [
```

Comment:
The way you designed this seems to have made your life more difficult downstream. (using Object.keys/values)

This might make it cleaner:

```
const GRAPHQL_COLUMNS = [
  { id: "title",
name: "Title" },
  { id: "description",
name: "Description" },
  { id: "jiraId",
name: "Jira ID" },
...
];
```

Then `Object.values(columnName)` becomes `columnName['name']`
`Object.keys(columnName)[0]` becomes `columnName['id']`